src/commands/pickFuncProcess.ts

```diff
@@ -9,7 +9,9 @@ import * as vscode from 'vscode';
 import { localize } from '../localize';
 import { funcHostTaskId } from './createNewProject/IProjectCreator';
 
-export async function pickFuncProcess(): Promise<string | undefined> {
+export async function pickFuncProcess(outputChannel: vscode.OutputChannel): Promise<string | undefined> {
+    outputChannel.show();
+    outputChannel.appendLine(localize('searchingFuncHost', 'Searching for Azure Functions host process...'));
```

Comment:
Cool.  Perhaps consider logging an issue against vscode to indicate sooner in the UI that something's happening, too.